sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/parameterized_spark.py

```diff
@@ -16,23 +16,24 @@
 class ParameterizedSpark(SparkJobEntryMixin):
     """Spark component that contains supporting parameters.
 
-    :param code: The source code to run the job.
+    :param code: The source code to run the job. Can be a local path or "http:", "https:", or "azureml:" url pointing
+        to a remote location.
     :type code: str
-    :param entry: Entry.
-    :type entry: str
-    :param py_files: List of .zip, .egg or .py files to place on the PYTHONPATH for Python apps.
+    :param entry: The file or class entry point.
```

Comment:
What is an entry point in this context? Also, why is the type changing dramatically from a string to a string dict? Based on the name I would've expected it to remain a string.

Reply:
- Spark components are configured to make a Spark container inside which the user can install their files and run their code. The entry point is the industry keyword for a script executable to run when the container starts that will let you run the container as if it were that binary.
 - I updated it to match the type in the actual constructor. Seems like whoever initially wrote the docstring didn't confirm it with the actual types or the types were updated after without updating the docstring.